src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
it's bad practice to reset such values in the `*` selector. The only thing allowed to set here is `box-sizing`